scripts/battery.sh

```diff
@@ -108,15 +108,16 @@ battery_status()
 
 main()
 {
+  bat_label=$(get_tmux_option "@dracula-battery-label" "♥")
   bat_stat=$(battery_status)
   bat_perc=$(battery_percent)
 
   if [ -z "$bat_stat" ]; then # Test if status is empty or not
-    echo "♥ $bat_perc"
+    echo "$bat_label $bat_perc"
   elif [ -z "$bat_perc" ]; then # In case it is a desktop with no battery percent, only AC power
-    echo "♥ $bat_stat"
+    echo "$bat_label $bat_stat"
   else
-    echo "♥ $bat_stat $bat_perc"
+    echo "$bat_label $bat_stat $bat_perc"
```

Comment:
Why are these changed?

Reply:
Hi @ethancedwards8 . I removed the unnecassary changes in my branch. There was a wrong git pull. This task is fixed.